hal/hal-bitstreams-p0.json

```diff
@@ -0,0 +1,115 @@
+{
+  "_links": {
+       "self": { "href": "/bitstreams{&sort,page,size}", "templated": true },
+       "next": { "href": "/bitstreams?page=1&size=5{&sort}", "templated": true },
+       "find": { "href": "/bitstreams/{id}", "templated": true }
+     },
+  "_embedded": [
+    {
+      "_links": {
+             "self": { "href": "/bitstreams/43c57c2b-206f-4645-8c8f-5f10c84b09fa" },
+             "bundle": { "href": "/bundles/35e0606d-5e18-4f9c-aa61-74fc751cc3f9" },

```

Comment:
this is link is probably wrong (it was wrong also in the initial sample).
My guess is that we should have a relative link here like /bitstreams/43c57c2b-206f-4645-8c8f-5f10c84b09fa/bundle as this represent the relation bitstream-bundle and not yet the linked bundle. If we want to embed the information of the bundle directly in the response we need to add a nested _embedded section with 
`_embedded:  [ 
    "bundle": {
           "_links": 
                   { "self" : { "href": "/bundles/35e0606d-5e18-4f9c-aa61-74fc751cc3f9" }
     }]`